routes/utils/errors.js

```diff
@@ -47,5 +47,9 @@ function parse (error) {
     error.message = 'Only admins can access /users'
   }
 
+  if (error.message === 'Document update conflict') {
+    error.message = 'An account with that username already exists'
+  }
+
```

Comment:
no Problem Eliška, we are looking forward to your update :) Ping us if you have any questions